diff --git a/data/resources/ui/pages/disk.blp b/data/resources/ui/pages/disk.blp
index 733760c..01d4a8a 100644
--- a/data/resources/ui/pages/disk.blp
+++ b/data/resources/ui/pages/disk.blp
@@ -32,7 +32,8 @@ template $DiskPage : Stack {
           valign: end;
           use-underline: true;
           clicked => $clicked_disks_button();
-          styles ["pill"]
+          styles ["pill"];
+          visible="false"
         }
       }
     };
diff --git a/data/resources/ui/pages/partition.blp b/data/resources/ui/pages/partition.blp
index aa52205..65308f6 100644
--- a/data/resources/ui/pages/partition.blp
+++ b/data/resources/ui/pages/partition.blp
@@ -35,109 +35,6 @@ template $PartitionPage : Stack {
           }
         }
       }
-
-      Stack partition_stack {
-        vhomogeneous: false;
-        transition-type: crossfade;
-
-        StackPage {
-          name: "available";
-          child: Box {
-            orientation: vertical;
-            spacing: 12;
-
-            Label {
-              /*Translators: Heading of list of partitions*/
-              label: _("Partitions");
-              justify: center;
-              wrap: true;
-              styles ["heading"]
-            }
-
-            ListBox partition_list {
-              receives-default: true;
-              valign: start;
-              row-activated => $use_partition();
-              styles ["boxed-list"]
-            }
-          };
-        }
-
-        StackPage {
-          name: "no-partitions";
-          /* TODO: could probably use some more love */
-          child: Label {
-            valign: start;
-            justify: center;
-            label: _("Disk has no partitions.");
-          };
-        }
-
-        StackPage {
-          name: "no-boot-partition";
-          child: Box {
-            margin-start: 18;
-            margin-end: 18;
-            margin-top: 6;
-            margin-bottom: 6;
-            spacing: 18;
-            halign: center;
-
-            Image {
-              icon-name: "dialog-information-symbolic";
-              styles ["dim-label"]
-            }
-
-            Box {
-              hexpand: true;
-              orientation: vertical;
-
-              Label {
-                /* Translators: Shown if a disk is currently not compatible with the way a system was booted. */
-                label: _("Selecting a partition is not possible.");
-                wrap: true;
-                xalign: 0;
-                styles ["dim-label"]
-              }
-
-              Label {
-                /* Translators: Followed by a list of requirements */
-                label: _("The disk requires:");
-                wrap: true;
-                xalign: 0;
-                styles ["dim-label"]
-              }
-
-              Stack {
-                hhomogeneous: false;
-                vhomogeneous: false;
-
-                StackPage {
-                  name: "efi";
-                  child: Label {
-                    /* Translators: Bullet point listing. */
-                    label: _("  • GPT partitioning\n  • EFI partition");
-                    wrap: true;
-                    xalign: 0;
-                    styles ["dim-label"]
-                  };
-                }
-
-                StackPage {
-                  name: "bios";
-                  child: Label {
-                    /* Translators: Bullet point listing. */
-                    label: _("  • BIOS boot partition");
-                    wrap: true;
-                    xalign: 0;
-                    styles ["dim-label"]
-                  };
-                }
-              }
-            }
-          };
-        }
-      }
     };
   }
 
