diff --git a/data/resources/ui/pages/summary.blp b/data/resources/ui/pages/summary.blp
index 633a86a..cff8b1f 100644
--- a/data/resources/ui/pages/summary.blp
+++ b/data/resources/ui/pages/summary.blp
@@ -51,6 +51,7 @@ template $SummaryPage : Box {
     Adw.ActionRow user_row {
       name: "user";
       activatable: true;
+      visible: false;
       /* Translators: Description of defined user account name and autologin status. */
       title: _("User Account");
       styles ["property"]
@@ -75,6 +76,7 @@ template $SummaryPage : Box {
     Adw.ActionRow format_row {
       name: "format";
       activatable: true;
+      visible: false;
       /* Translators: Description of selected format. */
       title: _("Formats");
       styles ["property"]
@@ -87,6 +89,7 @@ template $SummaryPage : Box {
     Adw.ActionRow timezone_row {
       name: "timezone";
       activatable: true;
+      visible: false;
       /* Translators: Description of selected timezone. */
       title: _("Timezone");
       styles ["property"]
@@ -100,6 +103,7 @@ template $SummaryPage : Box {
       margin-top: 6;
       name: "software";
       activatable: true;
+      visible: false;
 
       [prefix]
       Box {
@@ -149,6 +153,7 @@ template $SummaryPage : Box {
     Adw.ActionRow feature_row {
       name: "feature";
       activatable: true;
+      visible: false;
       /* Translators: Description of selected additional software. */
 
       [prefix]
